### DISCUSSION TITLE
Is Windows supported?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [hardinfo2/hardinfo2#64] [Q&A] [answered]
Whatsoever? It's listed at https://alternativeto.net/software/hardinfo2/?platform=windows as an alternative to a few Windows packages.

**Comment by hwspeedy** [comment, marked as answer]
Not as a native windows app - it cannot be compiled to windows.

But is works great with system information and benchmarks etc. under Windows Subsystem Linux 2. (WSL2).

https://learn.microsoft.com/en-us/windows/wsl/about#what-is-wsl-2
